### PR TITLE
feature: Add disabled state for selects

### PR DESCRIPTION
**Pull request by RED143**
image
image

### CODE REVIEW

**Review by hungsu — APPROVED** [review]
Thank you for cleaning up!

File: src/SelectSingle.vue [hunk 1]
```diff
@@ -56,7 +56,6 @@
 			// Potential props:
 			// https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select
 			// size
-			// disabled
```
Comment (jackkoppa) [inline src/SelectSingle.vue:59]: Nice great catch to even grab this documentation update!

**Review by jackkoppa — APPROVED** [review]
Looks excellent, @RED143! Thank you!

**Comment by RED143** [comment]
@jackkoppa, @hungsu  Could I merge this pr?

**Comment by hungsu** [comment]
Please do!

**Pull request merged**